components/icon-button.jsx

```diff
@@ -4,18 +4,23 @@ var Link = Router.Link;
 var ImageTag = require('./imagetag.jsx');
 
 var LinkSwap = React.createClass({
+  propTypes: {
+    linkTo: React.PropTypes.string,
+    href: React.PropTypes.string,
+    onClick: React.PropTypes.func
```

Comment:
yea and I think this means we can't just pass a URL as `href` to `<Link>`? :confused: dislike